docs/CHANGES.md

```diff
@@ -4,6 +4,7 @@
 
 - #585 Duplicate contact created due to JID case sensivity [saganshul]
 - #628 Fixes the bug in displaying chat status during private chat. [saganshul]
+- #704 Automatic fetching of registration form when registration_domain is set. [smitbose]
```

Comment:
`registration_domain` should be a hyperlink like I showed in my previous comment.